src/Package/Test/Repl/ReplCommandTest.cs

```diff
@@ -30,12 +31,14 @@ public class ReplCommandTest : IDisposable {
         private readonly VsDebuggerModeTracker _debuggerModeTracker;
         private readonly IRInteractiveWorkflow _workflow;
         private readonly IRInteractiveWorkflowProvider _workflowProvider;
+        private readonly TestCoreShell _coreShell = new TestCoreShell(null);
```

Comment:
We need constructor overload in `TestCoreShell` that will accept `IServiceContainer` as a parameter